changelog.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+#Unreleased
```

Comment:
This markdown is malformed:

![image](https://user-images.githubusercontent.com/1440090/42156663-9087899a-7de3-11e8-85a1-69920e395286.png)

Reply:
It also doesn't indicate to the next developer which type of change it is ... i.e. major / minor / patch

Reply:
The markdown plugin displayed it correct. Will fix this.